core/src/com/projectkorra/projectkorra/ability/WaterAbility.java

```diff
@@ -354,6 +430,22 @@ public static void playWaterbendingSound(final Location loc) {
 		}
 	}
 
+	public static boolean updateSourceBlock(final Block sourceBlock) {
+		if (isCauldron(sourceBlock)) {
+			GeneralMethods.setCauldronData(sourceBlock, ((Levelled) sourceBlock.getBlockData()).getLevel() - 1);
+			return true;
+		} else if (isTransformableBlock(sourceBlock) && !isCauldron(sourceBlock)) {
```

Comment:
```suggestion
		} else if (isTransformableBlock(sourceBlock)) {
```
!isCauldron is always true here because that is the first check of the if